src/database/schema/ChatSchema.ts

```diff
@@ -355,8 +405,10 @@ class ChatSchema implements BaseDbSchema {
         response?.message?.updated_at,
         JSON.stringify(response),
         response?.message?.id,
+        JSON.stringify(response?.message?.attachments),
         this.id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` is updated in the database without checking if it exists or not. This could potentially lead to errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->